content/en/tracing/guide/setting_up_apm_with_kubernetes_service.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Setting up APM with Kubernetes Service
+kind: guide
+further_reading:
+- link: "/containers/kubernetes/apm/"
+  tag: "Documentation"
+  text: "Set up trace collection"
+- link: "/containers/cluster_agent/admission_controller"
+  tag: "Documentation"
+  text: "Admission Controller"
+---
+
+## Overview
+
+There are 3 main ways the Datadog Tracers send their data to the Datadog Agent for the proper handling. Those are: Host IP, Unix Domain Socket (UDS), and the Kubernetes service. These options are all setup in a way to ensure the APM data the application pod sends arrive to the Datadog Agent pod on the same node. This strategy is meant to properly balance the traffic and ensure the correct tagging of your data.
```

Comment:
```suggestion
Datadog tracers can send data to the Datadog Agent in three ways: Unix Domain Socket (UDS), host IP, or a Kubernetes service. Each option ensures that when an application pod sends APM data to the Datadog Agent pod, all of the data arrives on the same node.
This strategy is meant to properly balance traffic and ensure the correct tagging of your data.
```

Reply:
Pretty much added this suggestions with like 1-2 changes